streams/test-utils/src/main/java/org/apache/kafka/streams/processor/api/MockProcessorContext.java

```diff
@@ -336,6 +344,14 @@ public void setRecordMetadata(final String topic,
         recordMetadata = new MockRecordMetadata(topic, partition, offset);
     }
 
+    public void setCurrentSystemTimeMs(final long currentSystemTimeMs) {
+        this.currentSystemTimeMs = currentSystemTimeMs;
+    }
+
+    public void setCurrentStreamTimeMs(final long currentStreamTimeMs) {
+        this.currentStreamTimeMs = currentStreamTimeMs;
+    }
```

Comment:
This wasn't in the KIP... Do we need it for something?

Reply:
I see. I have updated the KIP and the vote thread 👍🏽